stock_picking_auto_create_lot/models/stock_move_line.py

```diff
@@ -1,39 +1,12 @@
-# Copyright 2020 ACSONE SA/NV
+# Copyright 2024 Quartile Limited
```

Comment:
```suggestion
# Copyright 2020 ACSONE SA/NV
# Copyright 2024 Quartile Limited
```
Please don't remove this.

Reply:
It is already removed in OCA because it is removed this file first in OCA repo by another PR that is proposed by other community member. Then, we added this file again because it is needed for some use case. So, I guess we don't need add in our repo as well.

Reply:
@AungKoKoLin1997 
I understood.